src/shared/table/TableCell.js

```diff
@@ -17,7 +17,8 @@ const TableCell = (props) => {
     const langModalObj = {
       framework: allHeaders.find((h) => h.name === framework).name,
       category: catObj.row_title,
-      original_lang: catObj.original_lang
+      original_lang: catObj.original_lang,
+      link: allHeaders.find((h) => h.name === framework).url
```

Comment:
Since we're already doing this `allHeaders.find` thing, can we save that as a const outside of the langModalObj and then reference it for the framework & url? So something like this:

```jsx
const handleClick = (e) => {
 const frameworkObj = allHeaders.find((h) => h.name === framework)
const langModalObj = {
framework: framework.name,
link: framework.url,
category: catObj.row_title,
original_lang: catObj.original_lang
}
}
```